memphis/consumer.py

```diff
@@ -91,16 +97,29 @@ async def main():
         self.t_consume = asyncio.create_task(self.__consume(callback))
 
     async def __consume(self, callback):
-        subject = get_internal_name(self.station_name)
-        consumer_group = get_internal_name(self.consumer_group)
-        self.psub = await self.connection.broker_connection.pull_subscribe(
-            subject + ".final", durable=consumer_group
-        )
+        if not self.inner_station_name in self.connection.partition_consumers_updates_data:
```

Comment:
Please change to `if self.inner_station_name not in self.connection.partition_consumers_updates_data`